gensrc/thrift/PlanNodes.thrift

```diff
@@ -116,7 +116,7 @@ struct TBrokerRangeDesc {
     // total size of the file
     8: optional i64 file_size
     // number of columns from file
-    9: optional i32 num_of_columns_from_file = 0
+    9: optional i32 num_of_columns_from_file
```

Comment:
Because if code set the default value here, thrift's __isset will default to true, there's no way to determine the compatibility of fe/be, and the default value here of 0 doesn't make any sense, so I removed it